sdk/inc/azure/core/az_version.h

```diff
@@ -29,6 +29,6 @@
 #define AZ_SDK_VERSION_PATCH 0
 
 /// Optional pre-release identifier. SDK is in a pre-release state when present.
-#define AZ_SDK_VERSION_PRERELEASE "preview.6"
+#undef AZ_SDK_VERSION_PRERELEASE
```

Comment:
For Doxygen:
```suggestion
#define AZ_SDK_VERSION_PRERELEASE
#undef AZ_SDK_VERSION_PRERELEASE
```

Reply:
This is still not ideal. The docs will look as if it is defined. But if we go and undef it without ever defining, the info on AZ_SDK_VERSION_PRERELEASE will be missing from the docs.

Reply:
Where do you see that?
I don't see `AZ_SDK_VERSION_PRERELEASE` anymore in the docs from the generated artifacts:
![image](https://user-images.githubusercontent.com/6527137/93837128-e09de100-fc39-11ea-9d06-cb1867a6f84b.png)

Reply:
With this suggestion (define then undef next line), you will be able to:
![image](https://user-images.githubusercontent.com/41349689/93837494-1becdf80-fc3b-11ea-9ebd-f4ac5df91d76.png)
![image](https://user-images.githubusercontent.com/41349689/93837515-2d35ec00-fc3b-11ea-92ac-6a7b3f552788.png)

Reply:
It might be better that it is hidden and not visible in the docs, which we get by jus keeping undef. But your call Rick.

Reply:
If we don't want in documentation, I suggest to wrap the whole thing, including the doxygen comment, as 
```
/*
/// Optional pre-release identifier. SDK is in a pre-release state when present.
#define AZ_SDK_VERSION_PRERELEASE "beta.1"
*/
``` 
- this way, note that it sets the format, and there is not Doxygen comment
If we have a doxygen comment, and then an undef entry but without a #define, the doxygen comment woud stickto the next documentable entity, if there's any in the future. But if we comment (note that we don't use the doxygen-style multi-line comment), then it is excluded alltogether.

Reply:
I see. In that case, define/undef, like you suggested makes sense.